cv_next/app/actions/cvs/uploadCv.ts

```diff
@@ -25,10 +25,12 @@ export const checkUploadCV = async ({
     return "Error getting user.";
   }
   const userId = connectedUser.data.user.id;
+  //added a check, why undefined?
```

Comment:
Who are you asking? 🤔 
If this is a team-related question, ask the relevant programmer (git blame). There's no point in leaving such a comment in the code :)

Reply:
got it, but you also didn't answer my question

Reply:
and the git blame shows @Aradon267